table-webview/webpack.config.js

```diff
@@ -0,0 +1,50 @@
+// @ts-check
```

Comment:
Why do we need a webpack configuration if this should be generic code? I would have thought that any webview implementation reusing this table view code would pack it.